dev/Repeater/ItemsRepeater.h

```diff
@@ -135,36 +109,18 @@ class ItemsRepeater :
 
     static winrt::DependencyProperty GetVirtualizationInfoProperty()
     {
+        static GlobalDependencyProperty s_VirtualizationInfoProperty =
```

Comment:
Why is this needed here?

Reply:
Because it's an internal attached property and was previously being registered along with all of repeaters' other properties in EnsureProperties. This maybe is a shortcoming of the DP codegen, these internal attached properties come up occasionally.